microsoftgraph/microsoft-graph-explorer-v2
# Feature: show query tip message

## Overview

Displays a tip message for queries with parameters that need changing before running the query.
Related to #270 
FIxes #297

### Demo


![image](https://user-images.githubusercontent.com/53429535/69548104-c78bbb80-0fa7-11ea-87b6-289f486f8bf4.png)


### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* Log in to Graph Explorer
* On the sidebar in the Groups section, select 'group members'
* Notice the query tip shows up with a tip on how to best run the query
